staging/src/k8s.io/kubectl/pkg/cmd/cmd_test.go

```diff
@@ -99,19 +100,19 @@ func TestKubectlSubcommandShadowPlugin(t *testing.T) {
 		},
 		{
 			name: "test that a plugin does not execute over an existing command by the same name",
-			args: []string{"kubectl", "version"},
+			args: []string{"kubectl", "version", "--client=true"},
```

Comment:
to assure that we are executing builtin because only builtin `version` command has `--client` flag?

Reply:
same here, in order not to need a cluster connection